firebaseFunctions/src/index.ts

```diff
@@ -238,10 +241,35 @@ export const CI = {
           return response.sendStatus(500);
         }
 
-        const signerAddress = verifyMessage(
-          message,
-          signature,
-        );
+        let signerAddress;
+        /**
+         * Verify the signature.
+         * There are 2 cases:
+         * 1. "Normal" signature
+         * 2. The WalletConnect Safe App uses EIP-1271
```

Comment:
Why do you say "WalletConnect"?  This is certainly needed when running as a Safe App, but that has nothing to do with WalletConnect.  Is it also needed when using a Safe Wallet from WalletConnect?  What about other wallets?